serve/graph/model/transaction.go

```diff
@@ -47,6 +47,26 @@ func (t *Transaction) BlockHeight() int {
 	return int(t.txResult.Height)
 }
 
+func (t *Transaction) ResponseError() string {
+	if t.txResult.Response.Error == nil {
+		return ""
+	}
+
+	return t.txResult.Response.Error.Error()
+}
+
+func (t *Transaction) ResponseLog() string {
+	return t.txResult.Response.Log
+}
+
+func (t *Transaction) ResponseInfo() string {
+	return t.txResult.Response.Info
+}
+
+func (t *Transaction) ResponseData() []byte {
+	return t.txResult.Response.Data
+}
```

Comment:
WDYT about having a Response GraphQL object?

Reply:
I agree with @ajnavarro .

How about a Response GraphQL object like this ?

```
"""
`TransactionResponse` is the processing result of the transaction.
It has `log`, `info`, `error`, and `data`.
"""
type TransactionResponse {
  """
  The log value associated with the Transaction execution, if any.
  """
  log: String!

  """
  The Info associated with the Transaction execution, if any.
  """
  info: String!

  """
  The error value associated with the Transaction execution, if any.
  """
  error: String!

  """
  The response data associated with the Transaction execution, if any.
  """
  data: String!
}
```

Reply:
Can you please review the implementation ?

https://github.com/gnolang/tx-indexer/pull/32/commits/c0f0cf3c964a4e1bcd0738cfd7254a010329a9cc#diff-3a6df2b898459c1183102e35be08aa58206aadf7c363a393dae0db228d781c1bR126-R149